config/config.json

```diff
@@ -1,9 +1,9 @@
 {
```

Comment:
Can you merge down stable into this dev branch? I think it would make these changes go away since stable has the config updated with the database info already. If that doesn't resolve these changes though that's not a big deal.